src/overlays/actors/ovl_En_Nb/z_en_nb.h

```diff
@@ -6,11 +6,84 @@
 
 struct EnNb;
 
+typedef void (*EnNbActionFunc)(struct EnNb*, GlobalContext*);
+typedef void (*EnNbDrawFunc)(struct EnNb*, GlobalContext*);
+
 typedef struct EnNb {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1DC];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[19];
+    /* 0x0202 */ Vec3s transitionDrawTable[19];
+    /* 0x0274 */ s16 eyeIdx;
+    /* 0x0276 */ s16 blinkTimer;
+    /* 0x0278 */ s32 action;
+    /* 0x027C */ s32 drawMode;
+    /* 0x0280 */ f32 alphaTimer;
+    /* 0x0284 */ u32 alpha;
+    /* 0x0288 */ s32 flag;
+    /* 0x028C */ s32 unk_28C; // some sort of action
+    /* 0x0290 */ f32 timer;
+    /* 0x0294 */ ColliderCylinder collider;
+    /* 0x02E0 */ s32 headTurnFlag;
+    /* 0x02E4 */ Vec3f initialPos;
+    /* 0x02F0 */ Vec3f finalPos;
+    /* 0x02FC */ s16 pathYaw;
+    /* 0x02FE */ u16 movementTimer;
+    /* 0x0300 */ struct_80034A14_arg1 struct_300;
 } EnNb; // size = 0x0328
 
+typedef enum {
+    /* 0x00 */ NB_TYPE_0,
+    /* 0x01 */ NB_TYPE_1,
```

Comment:
are these used at all? if not, maybe they could be removed from the enum

Reply:
I have them for completeness, and yeah maybe they can be removed but I'm not really sure. I'll wait for Roman or someone else to get second input on it